pkg/proxy/proxier.go

```diff
@@ -549,6 +641,37 @@ func (proxier *Proxier) iptablesHostPortalArgs(destIP net.IP, destPort int, prot
 	return args
 }
 
+// Build a slice of iptables args for a from-container public-port rule.
+// See iptablesContainerPortalArgs
+// TODO: Should we just reuse iptablesContainerPortalArgs?
```

Comment:
I'm inclined to say yes, but don't change that iuntil I have a concrete proposal for the addrtype idea above.